model/src/main/proto/javaclasses/mealorder/identifiers.proto

```diff
@@ -33,61 +33,78 @@ option java_generate_equals_and_hash = true;
 import "javaclasses/mealorder/values.proto";
 import "google/protobuf/timestamp.proto";
 
-// An ID of a user represented by User email address.
+// An ID of a user.
+//
+// An identifier is represented by User email address of the next type:
+// <Name>.<Surname>@teamdev.com
 //
 message UserId {
 
-    EmailAdress email = 1;
+    // User's e-mail.
+    EmailAddress email = 1;
 }
 
-// An ID of a vendor represented by Vendor name.
+// An ID of a vendor.
+//
+// An identifier is represented by vendor's name which is unique for each vendor.
 //
 message VendorId {
 
+    // A name of a vendor.
     string vendor_name = 1;
```

Comment:
It's either ID or a name. It cannot be both. IDs are shorter, and can be generated from names. You're likely to need both.